Sources/Public/CalendarView.swift

```diff
@@ -1034,7 +1043,14 @@ public final class CalendarView: UIView {
     }
 
     scrollMetricsMutator.applyOffset(autoScrollOffset)
-    updateSelectedDayRange(dragGestureRecognizer: multipleDaySelectionGestureRecognizer)
+
+    if multiDaySelectionLongPressGestureRecognizer.state != .possible {
+      updateSelectedDayRange(gestureRecognizer: multiDaySelectionLongPressGestureRecognizer)
+    } else if multiDaySelectionPanGestureRecognizer.state != .possible {
+      updateSelectedDayRange(gestureRecognizer: multiDaySelectionPanGestureRecognizer)
+    } else {
+      fatalError("The autoScrollDisplayLink should not fire if both gesture recognizers are in the `.possible` state.")
+    }
```

Comment:
the only real logic change - we need to see which of the two gesture recognizers is active, and pass that to the `updateSelectedDayRange` function.